test.sh

```diff
@@ -212,6 +212,37 @@ Bottom text$CR
 EOF
 ) | tee test/smtp_send_email \
   | diff <(printf "") /dev/stdin
+  #
+# Check that the user cannot login to matrix with an invalid orbit ID
+curl --url "https://$SINGULARITY_HOSTNAME/_matrix/client/r0/login" \
+  --unix-socket ./socks/https.sock \
+  --verbose \
+  --cacert test/ca_cert.pem \
+  --no-progress-meter \
+  --header "Content-Type: application/json" \
+  --data "{
+        \"type\": \"m.login.password\",
+        \"user\": \"@user:$SINGULARITY_HOSTNAME\",
+        \"password\": \"ssap\"
+      }" \
+  | tee test/matrix_login_invalid \
+  | grep '{"errcode":"M_FORBIDDEN","error":"Invalid username or password"}'
+
+# short sleep to avoid "429: Too Many Requests" error from synapse
+sleep 2
```

Comment:
can we go lower? like maybe a few hundred milliseconds would be enough

Reply:
sleep 1 was causing problems before but let me try it again

Reply:
sleep 1 is great now